src/components/StageBar.vue

```diff
@@ -38,31 +38,51 @@ export default {
 @import '@/styles/theme.scss';
 
 .stage-bar {
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: center;
-  padding-left: 16px;
-
+  position: relative;
+  display: grid;
```

Comment:
Each stage shouldn't have the same width. Maybe this behaviour will be easier to achieve with `flex`.